orders/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls    import path
+
+from orders.views    import OrderView
+
+urlpatterns = [
+    path(''                                    , OrderView.as_view()),
+    path('/<int:order_id>'                     , OrderView.as_view()),
+    path('/<int:order_id>/<int:order_item_id>' , OrderView.as_view()),
```

Comment:
```suggestion
    path('/<int:order_id>/order-items/<int:order_item_id>' , OrderView.as_view()),
```
이게 조금 더 RESTful한 것 같습니다.

Reply:
order_id만 받도록 urls.py 수정하였습니다.